### PR TITLE
Add spliffy framework

### PR DESCRIPTION
**Pull request by snowbldr**

### CODE REVIEW

File: javascript/spliffy/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@srfnstack/spliffy": "^0.7.0"
```
Comment (waghanza) [inline javascript/spliffy/package.json:3]: please use `~` to stick to the same minor version

```suggestion
    "@srfnstack/spliffy": "~0.7.0"
```

File: javascript/spliffy/app.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require( '@srfnstack/spliffy' )( {
+    routeDir: __dirname + '/www',
+    port: 3000,
```
Comment (waghanza) [inline javascript/spliffy/app.js:3]: 👍🏻

**Review by waghanza — CHANGES_REQUESTED** [review]
missing `config.yaml`

Thanks you so much for your contribution :heart:

File: javascript/spliffy/package.json (hunk 1 above)
Comment (snowbldr) [reply javascript/spliffy/package.json]: Updated. Thanks for checking.

File: javascript/spliffy/config.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+framework:
+  website: srfnstack.github.io/spliffy/
+  github: SRFNStack/spliffy
+  version: 0.7
+
```
Comment (waghanza) [inline javascript/spliffy/config.yaml:5]:
```suggestion

build_deps:
  - git
```

File: javascript/spliffy/config.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+build_deps:
+  - git
```
Comment (waghanza) [inline javascript/spliffy/config.yaml:7]:
```suggestion
build_deps:
  - git

bin_deps:
  - gcompat # required by uWebSocket
```

**Comment by snowbldr** [comment]
I built and ran the dockerfile locally, so it should be ok. It likely failed the last build because the www directory didn't exist.

**Review by waghanza — APPROVED** [review]

**Pull request merged**